curriculum/challenges/english/25-front-end-development/lab-stylized-to-do-list/66c051d13a6a20255a963695.md

```diff
@@ -0,0 +1,320 @@
+---
+id: 66c051d13a6a20255a963695
+title: Build a Stylized To-Do list
+challengeType: 14
+dashedName: build-a-stylized-to-do-list
+demoType: onClick
+---
+
+# --description--
+
+In this lab, you will practice the different styles that can be applied to links when they are hovered over, focused, clicked, and visited.
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab.
+
+**User Stories:**
+
+1. You should have an unordered list with the class `todo-list`.
+2. Inside the unordered list, you should have 4 list items.
+3. Inside each list item, there should be:
+   
+   - An `input` element with the type `checkbox` and `id` set to a unique value.
+   - A `label` element with the `for` attribute set to the corresponding `input` element's `id`.
+   - An unordered list with the class `sub-item`.
+   - A list item with an `anchor` element in it. The `anchor should have the class `sub-item-link`, a valid `href` value, and a `target` value that makes the link open in a new tab.
+  
+4. You shouldn't apply any list styles to the unordered lists.
```

Comment:
I'm not sure what this one is supposed to mean. Does it mean I should use `list-style: none;` or am I not supposed to change the default list-style. If so, I would reword this to something like "You should remove the default list style from your unordered lists."

And we could add a test for it if so. Also, is it supposed to apply to all the UL's? or just the main UL? I would make it so it's just the main one. "You should remove the default list style from your `.todo-list`"

Reply:
This instruction is supposed to remove the default style from the main unordered list. The campers can add any style to the other list inside and it is optional.

I have updated the instruction and added a new test.
It works for both:

```css
.todo-list {
    list-style: none;
}
```

```css
.todo-list {
    list-style-type: none;
}
```